docs/content/docs/connectors/table/elasticsearch.md

```diff
@@ -67,15 +67,17 @@ Connector Options
       <tr>
         <th class="text-left" style="width: 25%">Option</th>
         <th class="text-center" style="width: 8%">Required</th>
+        <th class="text-center" style="width: 8%">Forwarded</th>
```

Comment:
Can we come up with a better term than `forwarded` to make clear that the state becomes incompatible? `forwarded` is derived by the actual implementation but I do not think many users can relate the term.

Reply:
Any ideas? Can't really think of anything better TBH. @twalthr ?

At some point we'll also add a proper documentation page in SQL to explain the upgrade story, and we can link it directly from the header of this table

Reply:
I still like the name proposed in the FLIP. How about `mutable`? I agree that the header should be a link later.

Reply:
What about `restoreable`?

Reply:
migratable? or just make it clearer: "forward to json plan"?